tests/phpunit/tests/blocks/wpBlock.php

```diff
@@ -218,9 +218,12 @@ public function test_constructor_prepares_context_for_inner_blocks() {
 		$context       = array( 'unrequested' => 'not included' );
 		$block         = new WP_Block( $parsed_block, $context, $this->registry );
 
-		$this->assertCount( 0, $block->context );
+		$this->assertCount( 1, $block->context );
 		$this->assertSame(
-			array( 'core/recordId' => 10 ),
+			array(
+				'unrequested'   => 'not included',
```

Comment:
This isn't working as intended. If the block doesn't define `unrequested` in the `uses_context`, it should not be exposed to the block.

Reply:
As we copied the `$this->available_context` value in `$this->context` these change needed as block registration use that context see https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/blocks/wpBlock.php#L218-L219